conf/locale/cy/LC_MESSAGES/django.po

```diff
@@ -2431,10 +2431,9 @@ msgstr "sicrhau bod eich dirwy yn seiliedig ar eich amgylchiadau"
 msgid "help to speed up the processing of your case"
 msgstr "helpu i gyflymu'r broses o ran hwyluso eich achos"
 
-# TODO
 #: apps/result/management/commands/process_results.py:96
 msgid "Make a plea result"
-msgstr "[[Welsh translation needed]]"
```

Comment:
This change is breaking the tests

https://travis-ci.org/ministryofjustice/manchester_traffic_offences_pleas/builds/224362695#L671

I think we need to update this line:
https://github.com/ministryofjustice/manchester_traffic_offences_pleas/blob/master/apps/result/tests.py#L444